From 55aac754214ac586105d0e74667e7a6359935ac6 Mon Sep 17 00:00:00 2001
From: Peter Bloomfield <PeterBloomfield@bellsouth.net>
Date: Tue, 5 Jun 2018 12:02:34 -0400
Subject: [PATCH] address-book-rubrica: Fix a typo

* libbalsa/address-book-rubrica.c (extract_net): Fix a typo that
  broke the build.
---
 ChangeLog                       | 5 +++++
 libbalsa/address-book-rubrica.c | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/libbalsa/address-book-rubrica.c b/libbalsa/address-book-rubrica.c
index 6b7b1b267..3b35750b5 100644
--- a/libbalsa/address-book-rubrica.c
+++ b/libbalsa/address-book-rubrica.c
@@ -684,7 +684,7 @@ extract_net(xmlNodePtr entry, GList ** mail_addrs)
 	if (!xmlStrcmp(entry->name, CXMLCHARP("Uri"))
 	    && g_strcmp0(uri_type = xml_node_get_attr(entry, CXMLCHARP("type")),
                          "email") == 0
-	    && (mail_addr = xml_node_get_text(entry) != NULL))
+	    && (mail_addr = xml_node_get_text(entry)) != NULL)
 	    *mail_addrs = g_list_prepend(*mail_addrs, mail_addr);
 	g_free(uri_type);
 
-- 
2.17.1

